model/src/main/kotlin/VcsInfo.kt

```diff
@@ -123,9 +122,7 @@ private class VcsInfoDeserializer : StdDeserializer<VcsInfo>(VcsInfo::class.java
         return VcsInfo(
             VcsType(node["type"].textValueOrEmpty()),
             node["url"].textValueOrEmpty(),
-            // For backward compatibility, if resolved_revision is set, prefer it over revision because it is more
-            // specific.
-            node["resolved_revision"]?.textValue() ?: node["revision"].textValueOrEmpty(),
+            node["revision"].textValueOrEmpty(),
```

Comment:
Would it make sense to fail hard if user is still using the old (now unhandled) format? That would be nice, in particular if it was possible to remove the custom serializer as per @sschuberth 's comment above.